### PR TITLE
[EDU-2597] Add new Main Settings page

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Added new main settings page
- Fixed links to direct to new page

TODO: Add to reference menu and adjust redirect from HTTP2 guide (deprecated)

### Additional links

[EDU-2597]: https://aziontech.atlassian.net/browse/EDU-2597?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx [hunk 1]
```diff
@@ -136,7 +136,7 @@ curl --location --request PATCH 'https://api.azionapi.net/edge_applications/<edg
   | `http_port` | Sets new HTTPS ports to the application. In this case, the array `[80, 8008]` defines that both ports 80 and 8008 can be used by HTTP requests. |
   | `https_port` | Sets new HTTPS ports to the application. In this case, the array `[443, 8443]` defines that both ports 443 and 8443 can be used by HTTP requests. |
   | `minimum_tls_version` | When set to `tls_1_2`, establishes that the minimum TLS version should be 1.2. |
-  | `supported_ciphers` | Sets the supported cipher suite, with its value in this case being `TLSv1.2_2021`. See the full list of [list of supported ciphers](/en/documentation/products/edge-application/#ciphers) for further details.
+  | `supported_ciphers` | Sets the supported cipher suite, with its value in this case being `TLSv1.2_2021`. See the full list of [list of supported ciphers](/en/documentation/products/edge-application/delivery-settings/#ciphers) for further details.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/build-journey/edit-edge-app/edit-main-settings/main-settings.mdx:139]:
```suggestion
  | `supported_ciphers` | Sets the supported cipher suite, with its value in this case being `TLSv1.2_2021`. See the full list of [list of supported ciphers](/en/documentation/products/edge-application/main-settings/#ciphers) for further details.
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/main-settings.mdx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+## Minimum TLS version
+
+The **Transport Layer Security (TLS)** protocol allows you to encrypt web traffic. The following TLS versions can be used with edge applications:
+
+- TLS 1.0 (*deprecated*)
+- TLS 1.1 (*deprecated*)
+- TLS 1.2
+- TLS 1.3
+
+You can choose the minimum version of TLS that will be supported by your edge application. By choosing recent versions of the protocol, older devices or browsers might not be able to access the edge application.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/main-settings.mdx:62]:
```suggestion
You can choose the minimum version of TLS that'll be supported by your edge application. By choosing recent versions of the protocol, older devices or browsers might not be able to access the edge application.
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/main-settings.mdx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+Azion blocks TLS Renegotiation and TLS Resumption by default. If you want to customize this setup, [contact the Sales team](https://www.azion.com/en/contact-sales/).
+
+---
+
+## TLS Ciphers
+
+Ciphers are cryptography algorithms utilized to encrypt *plaintext* into *ciphertext*, which requires a key to be decrypted. Azion gives you the possibility to change the *cipher suite* your edge application will use in order to protect your application against TLS attacks.
+
+The cipher suite will determine which cryptographic algorithms will be used in the TLS connections of your edge application. Both, client and server, will *negotiate* the cipher suite to securely encrypt and decrypt the data exchanged during the session.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/main-settings.mdx:72]:
```suggestion
The cipher suite will determine which cryptographic algorithms will be used in the TLS connections of your edge application. Both client and server will *negotiate* the cipher suite to securely encrypt and decrypt the data exchanged during the session.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/main-settings.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+permalink: /documentacao/produtos/edge-application/main-settings/
+---
+
+Com [Edge Application](/pt-br/documentacao/produtos/edge-application/), você pode escolher o protocolo de entrega e as portas de sua aplicação. Você também pode garantir a segurança da conexão HTTPS entre a aplicação e seus usuários com cifras TLS.
+
+---
+
+## Protocolos de entrega
+
+Você pode customizar a entrega de sua aplicação através das seguintes opções:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/main-settings.mdx:15]:
```suggestion
Você pode personalizar a entrega de sua aplicação através das seguintes opções:
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/main-settings.mdx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+- **HTTP**: entrega sua aplicação usando apenas o protocolo HTTP.
+- **HTTP & HTTPS**: entrega sua aplicação usando os protocolos HTTP e HTTPS.
+
+:::note
+Se você escolher entregar sua aplicação usando HTTP & HTTPS, você deve configurar [Digital Certificates](/pt-br/documentacao/produtos/edge-application/digital-certificates/) para seu domínio.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/main-settings.mdx:21]: configurar UM digital certificate?

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/main-settings.mdx [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+Por padrão, a Azion bloqueia *TLS Renegotiation* e *TLS Resumption*. Se você deseja personalizar essa configuração, entre em contato com nossa [equipe de Vendas](https://www.azion.com/pt-br/contate-vendas/).
+
+---
+
+## Cifras
+
+Cifras são algoritmos de criptografia utilizados para criptografar *texto simples* (`plaintext`) em *texto cifrado* (`ciphertext`), que requer uma chave para ser descriptografado. A Azion oferece a possibilidade de alterar o grupo de *cifras* que sua edge application usará, a fim de proteger sua aplicação contra ataques TLS.
+
+O grupo de cifras determinará quais algoritmos de criptografia serão utilizados nas conexões TLS da sua edge application. Ambos, cliente e servidor, *negociarão* o grupo de cifras para criptografar e descriptografar os dados trocados durante a sessão, garantindo a confidencialidade deles.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/main-settings.mdx:74]:
```suggestion
O grupo de cifras determinará quais algoritmos de criptografia serão utilizados nas conexões TLS da sua edge application. Ambos cliente e servidor *negociarão* o grupo de cifras para criptografar e descriptografar os dados trocados durante a sessão, garantindo a confidencialidade deles.
```

**Review by hannah-sk — APPROVED** [review]
deixei algumas sugestões

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/main-settings.mdx (hunk 5 above)
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/main-settings.mdx:21]:
```suggestion
Se você escolher entregar sua aplicação usando HTTP & HTTPS, você deve associar um [digital certificate](/pt-br/documentacao/produtos/edge-application/digital-certificates/) a seu domínio.
```

**Pull request merged**